### ISSUE TITLE
Build fails on Windows 10

### ISSUE THREAD

**Issue by jamesjessian** [patr0nus/Pym#2]
Unfortunately, it doesn't seem like squoosh is a good web app to be using as a cross-OS example for Deskgap at the moment, because it doesn't build on Windows.
I cloned and ran the build but I get the same...
`SyntaxError: Invalid regular expression: /\codecs\/: \ at end of pattern`
...error that these people are getting over in the squoosh repo:
https://github.com/GoogleChromeLabs/squoosh/issues/465
Looks like Jake Archibald is working on it on a [branch](https://github.com/GoogleChromeLabs/squoosh/compare/windows-build-fix).

I also can't install the Windows App from the store because it requires the Windows Insider/Review Release (v17763.1). I only have "17134.590" which I think is the latest Win 10 Pro update.

I guess I'm just bringing it to your attention and eager to see an example Deskgap app.

**Comment by wan9chi** [comment]
Yeah I encountered this error, too. You just have to build it on Linux and copy the result back to Windows.

The latest version of Windows 10 **is** v17763. Pym wouldn't be allowed in the Windows Store if it requires a insider release. Sometimes Windows Update won't work. You need [Update Assistant](https://support.microsoft.com/en-us/help/3159635/windows-10-update-assistant) to get you to the latest version.
